src/parsers/eventChecker.ts

```diff
@@ -0,0 +1,33 @@
+import {
```

Comment:
All tests should be in `filename.test.ts`

Reply:
I extracted to the chekcer because spies don't work on inner exported calls